egs/pashto/v1/local/fix_data_dir.sh

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
```

Comment:
is this copied from another v1 recipe? If yes, please make a note saying from where it's copied.